.github/workflows/pr.yml

```diff
@@ -0,0 +1,25 @@
+name: PR
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
We could probably just pull down a public open JDK 8 docker image instead of running on this vanilla VM and then skip the `actions/setup-java@v1` step below.

Reply:
That's a fair idea -- I've not seen any github actions posts that mention doing that and I suspect that `ubuntu-latest` is locally cached so the fetch is quick.

Takes ~7s to install java.